app.py

```diff
@@ -158,36 +205,104 @@ def show_column(dstats, display_list, show_perplexities, column_id=""):
     """
 
     # start showing stuff
-    st_utils.expander_header(dstats, DATASET_NAME_TO_DICT)
-    for widget_tuple in display_list:
+
+
+    tab1, tab2, tab3, tab4, tab5, tab6, tab7 = st.tabs(["Dataset Description", "General Text Statistics", "Label Distribution", "Text Lengths","Text Duplicates","nPMI","Zipfs Law Fit"])#,"Text Embedding Clustering"])
+    
+    with tab1:
+        logs.info("showing header")
+        st_utils.expander_header(dstats, DATASET_NAME_TO_DICT)
+  
+        
+    ##TO:DO @Ezi --> incorperate 
+    
+    i =1
+    for (widget_tuple, tab) in zip(display_list,[tab2, tab3, tab4, tab5, tab6, tab7]) :
         widget_type = widget_tuple[0]
         widget_fn = widget_tuple[1]
-        logs.info("showing %s." % widget_type)
-        try:
-            widget_fn(dstats, column_id)
-        except Exception as e:
-            logs.warning("Jk jk jk. There was an issue with %s:" % widget_type)
-            logs.exception(e)
+        with tab:
+            #tab_no = tabs
+            logs.info("showing %s" % widget_type)
+            try:
+                widget_fn(dstats, column_id)
+            except Exception as e:
+                logs.warning("Jk jk jk. There was an issue with %s:" % widget_type)
+                logs.exception(e)
+        i=i+1
+
+             
     # TODO: Fix how this is a weird outlier.
     if show_perplexities:
         st_utils.expander_text_perplexities(dstats, column_id)
     logs.info("Have finished displaying the widgets.")
+    # TODO: Fix how this is a weird outlier.
+
+  
+
 
 def main():
+    with open("style.css") as f:                                                # Reading style.css file and opening it 
+        st.markdown(f'<style>{f.read()}</style>',unsafe_allow_html= True)       # Storing all styles in the streamlit markdown  &   unsafe_allow_html to true so that we can use html tags with the code  
+    #----read and run html file
+
     parser = argparse.ArgumentParser()
     parser.add_argument(
         "--live", default=False, required=False, action="store_true", help="Flag to specify that this is not running live.")
     parser.add_argument(
         "--pull_cache_from_hub", default=False, required=False, action="store_true", help="Flag to specify whether to look in the hub for measurements caches. If you are using this option, you must have HUB_CACHE_ORGANIZATION=<the organization you've set up on the hub to store your cache> and HF_TOKEN=<your hf token> on separate lines in a file named .env at the root of this repo.")
     arguments = parser.parse_args()
     live = arguments.live
+
+
+
+    # Sidebar description and selection
+
+    st.title("Data Measurements Tool")
+    st.markdown("""
+    This demo showcases the [dataset metrics as we develop them](https://huggingface.co/blog/data-measurements-tool).
+    
+    With this tool, you can:
+    
+    ✓ Dynamic loading of datasets in the lib
```

Comment:
These checks look different than in the design, where instead they have a circle around them. Is it possible to make those instead?